Formula/gz-transport13.rb

```diff
@@ -0,0 +1,71 @@
+class Gztransport13 < Formula
```

Comment:
brew complaint: 

~~~
Expected to find class GzTransport13, but only found: Gztransport13.
~~~

Reply:
[5f8c284](https://github.com/osrf/homebrew-simulation/pull/2196/commits/5f8c2842ce862bab4545de8af59f6157d1e225cc)